modules/dr-restoring-cluster-state.adoc

```diff
@@ -265,7 +297,7 @@ $ sudo crictl ps | grep etcd | egrep -v "operator|etcd-guard"
 3ad41b7908e32       36f86e2eeaaffe662df0d21041eb22b8198e0e58abeeae8c743c3e6e977e8009                                                         About a minute ago   Running             etcd                                          0                   7c05f8af362f0
 ----
 
-.. From the recovery host, verify that the etcd pod is running.
+.. From the recovery host, verify that the etcd pod is running by running the following command:
```

Comment:
Backticks for `etcd`?

Reply:
In this case, as it refers to the etcd pod, it would make sense. But not as a general guideline on how to write "etcd" in the docs.